DeviceAdministration/Infrastructure/BusinessLogic/DeviceLogicND.cs

```diff
@@ -972,7 +1042,7 @@ private static IEnumerable<DevicePropertyMetadata> GetDevicePropertyConfiguratio
             };
         }
 
-        private async Task ValidateDevice(dynamic device)
+        private async Task ValidateDevice(DeviceND device)
```

Comment:
Same reason. ValidateDevice is only referred from Add Device. Didn't have to copy.